### ISSUE TITLE
Can't zoom

### ISSUE THREAD

**Issue by T8RIN** [pastthepixels/FreePaint#3]
While testing the previous issue i zoomed the scene out, and after app restart i can't zoom it in even for two pixels

![Screenshot_20230903-191024_FreePaint.png](https://github.com/pastthepixels/FreePaint/assets/52178347/6b1c7840-dcd4-4fac-9457-099e7dc17fed)

**Comment by T8RIN** [comment]
Sorry, my fault, it zooms, but the picture is 1*1 pixel and i didn't set that by hands

**Comment by pastthepixels** [comment]
So when you would go to the settings FreePaint told you this was 1px by 1px? This might be a duplicate of #2 then. FreePaint would have parsed the SVG as being 1px by 1px and saved the last used SVG size as a preference (usually intended so when you open an SVG of some size and restart FreePaint, you can create a new one with the same size).

**Comment by T8RIN** [comment]
> So when you would go to the settings FreePaint told you this was 1px by 1px? This might be a duplicate of #2 then. FreePaint would have parsed the SVG as being 1px by 1px and saved the last used SVG size as a preference (usually intended so when you open an SVG of some size and restart FreePaint, you can create a new one with the same size).

Maybe :(

**Comment by pastthepixels** [comment]
Even though the underlying issue here makes this a duplicate of #2, I've decided that this does raise an issue in need of fixing; that zoom limits make working on small documents difficult. I submitted a PR, #7, that should make opening and working with small documents easier by letting you zoom in/out however much you wanted.

That's pretty much a wrap with this issue then. Changes should show up in the next release of FreePaint!
